src/community/advanced-data-card.tsx

```diff
@@ -87,44 +90,86 @@ const CardContent = ({
     const {textPresets} = useTheme();
 
     return (
-        <Stack space={4}>
-            {headline}
-            {pretitle && (
-                <Text2
-                    color={vars.colors.textPrimary}
-                    truncate={pretitleLinesMax}
-                    as={pretitleAs}
-                    regular
-                    hyphens="auto"
-                >
-                    {pretitle}
-                </Text2>
+        /** using flex instead of nested Stacks, this way we can rearrange texts so the DOM structure makes more sense for screen reader users */
```

Comment:
AdvancedDataCard was missing the logic for the custom order of elements that other cards have